ironfish-cli/src/commands/service/stats.ts

```diff
@@ -0,0 +1,157 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { CurrencyUtils, PromiseUtils } from '@ironfish/sdk'
+import { WebApi } from '@ironfish/sdk'
+import { Flags } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+import { IronfishCliPKG } from '../../package'
+import { GossipForkCounter } from '../../utils/gossipForkCounter'
+
+export default class Stats extends IronfishCommand {
+  static hidden = true
+
+  static description = `Submits stats to telemetry API`
+
+  static flags = {
+    ...RemoteFlags,
+    endpoint: Flags.string({
+      char: 'e',
+      parse: (input: string) => Promise.resolve(input.trim()),
+      required: false,
+      description: 'API host to sync to',
+    }),
+    token: Flags.string({
+      char: 't',
+      parse: (input: string) => Promise.resolve(input.trim()),
+      required: false,
+      description: 'API host token to authenticate with',
+    }),
+    delay: Flags.integer({
+      required: false,
+      default: 60000,
+      description: 'Delay (in ms) to wait before uploading stats',
+    }),
+  }
+
+  async start(): Promise<void> {
+    const { flags } = await this.parse(Stats)
+
+    const apiHost = (
+      flags.endpoint ||
+      process.env.IRONFISH_API_HOST ||
+      'https://api.ironfish.network'
+    ).trim()
+
+    const apiToken = (flags.token || process.env.IRONFISH_API_TOKEN || '').trim()
+
+    const api = new WebApi({ host: apiHost, token: apiToken })
+
+    await this.sdk.client.connect()
+
+    // metric loops
+    void this.forks(api, flags.delay)
+    void this.feeRates(api, flags.delay)
+
+    // hold program open so that metrics can be sent
+    // eslint-disable-next-line no-constant-condition
+    while (true) {
+      await PromiseUtils.sleep(1000)
+    }
```

Comment:
I think an alternative to keep the process running might be to `await` the second metric loop